lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -52,8 +52,20 @@ module.exports.prototype = {
                 return;
             }
 
+            var prevToken = file.getPrevToken(comment, {
+                includeComments: true
+            });
+
+            if (prevToken.type === 'Line') {
+                return;
+            }
+
+            if (prevToken.type === 'Punctuator' && prevToken.value === '{') {
```

Comment:
Also, this isn't *really* correct as it allows inside Objects, which aren't blocks, but I suppose it's fine for now.